src/tactic/interactive.lean

```diff
@@ -630,8 +630,8 @@ add_tactic_doc
   tags       := ["testing"] }
 
 /--
-a weaker version of `trivial` that tries to solve the goal by reflexivity or by reducing it to true,
-unfolding only `reducible` constants. -/
+a weaker version of `trivial` that tries to solve the goal by reflexivity, finding
+contradictory hypotheses, or by reducing it to true, unfolding only `reducible` constants. -/
```

Comment:
It might be clearer mentioning contradiction last, since that's tried last, and also the `reducible` part doesn't apply to it. Here's a possibility derived from taking the `tactic.interactive.trivial` docstring and changing it for what `triv` does:
```suggestion
A weaker version of `trivial` that tries to solve the goal using a canonical proof of `true` or the `reflexivity` tactic
(unfolding only `reducible` constants, so can fail faster than `trivial`), and otherwise tries the `contradiction` tactic. -/
```

Reply:
What actually started me off on all this triv/trivial thing was that I was looking for a tactic which did *not* try `contradiction`! How about instead of changing the docstring ov `triv` I just change the tactic to do what it claims the docstring does?

Reply:
Spelunking into the git history, I guess `triv` exists for `use`, since `trivial` is too expensive for one of its automatic features.

Maybe `triv` could be renamed to `trivial'`, freeing up the name for the simpler version? Maybe it does `tactic.triv <|> tactic.reflexivity` so it's the beginning of `trivial`? (How's that for a mnemonic)

Reply:
Aah thank you! I will not fiddle with `triv` then.